components/casemap/data/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+[package]
+name = "icu_casemap_data"
+version = "1.3.0"
```

Comment:
I guess this should really be 1.2.0, but it isn't for any crate.

Reply:
Yeah I'm trying to be consistent so that when it's publishing time we can find-and-replace easily